zulipterminal/helper.py

```diff
@@ -347,3 +354,20 @@ def match_user(user: Any, text: str) -> bool:
         if keyword.startswith(text.lower()):
             return True
     return False
+
+
+@asynch
+def notify(title: str, text: str) -> None:
+    FNULL = open(os.devnull, 'w')
+    command = ""
+    if WSL:
+        command = ('powershell.exe "New-BurntToastNotification'
+                   ' -Text \'{}\', \'{}\'"'.format(title, text))
+    elif MACOS:
+        command = ("osascript -e 'display notification \"{}\" with title"
+                   " \"{}\"'".format(text, title))
+    elif LINUX:
+        command = 'notify-send {} {}'.format(title, text)
```

Comment:
This worked for me when both had `"{}"` instead, to ensure the command considers them as two arguments rather than (likely) many more.

Reply:
Thanks!